lib/fee_threshold.rb

```diff
@@ -11,14 +11,15 @@ class FeeThreshold
       { lower: 6001, upper: 7000, amount: 14000 }
     ]
 
-  def initialize(application)
-    @application = application
+  def initialize(fee)
+    return if fee.nil?
+    @fee = fee.round
   end
 
   def band
-    return if fee.nil?
-    return 3000 if fee <= 1000
-    return 16000 if fee >= 7001
+    return if @fee.nil?
```

Comment:
Writing this for historical reasons, as we've already discussed this in person.

By removing the nil guard in the `band` method, I would get bitten, see:

```
f = FeeThreshold.new(nil)
=> #<FeeThreshold:0x007fa469a387b0>
[4] pry(main)> f.band
=> nil
[5] pry(main)> f.class
=> FeeThreshold
```